tests/tests_suite_test.go

```diff
@@ -84,6 +85,11 @@ func (s *newSspStrategy) Init() {
 		return apiClient.Create(ctx, namespaceObj)
 	}, timeout, time.Second).ShouldNot(HaveOccurred())
 
+	Eventually(func() error {
+		namespaceObj := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: s.GetNewTemplatesNamespace()}}
+		return apiClient.Create(ctx, namespaceObj)
+	}, timeout, time.Second).ShouldNot(HaveOccurred())
```

Comment:
This namespace is only used in 1 test case. Can you move the code, so that this namespace is created and removed inside the test?